src/app/shared/profile/profile.component.ts

```diff
@@ -1,13 +1,22 @@
 import { Component, Input } from '@angular/core';
+import { formatDate } from '@angular/common';
 
 @Component({
     selector: 'app-profile',
     templateUrl: './profile.component.html',
     styleUrls: ['./profile.component.less'],
 })
 export class ProfileComponent {
+    _date: string;
     @Input() username?: string = '';
```

Comment:
Is it possible to be optional?
```suggestion
    @Input() username: string = '';
```